x-pack/build_chromium/darwin/args.gn

```diff
@@ -19,13 +19,13 @@ use_alsa = false
 use_cups = false
 use_dbus = false
 use_gio = false
-# Please, consult @elastic/kibana-security before changing/removing this option.
-use_kerberos = false
 use_libpci = false
 use_pulseaudio = false
 use_udev = false
 
 is_debug = false
 symbol_level = 0
 is_component_build = false
-remove_webcore_debug_symbols = true
```

Comment:
`remove_webcore_debug_symbols` caused warning logs during compilation: it is unused